### PR TITLE
Add msilib and _msi

### PR DESCRIPTION
**Pull request by CraftSpider**
Adds msilib stubs, and _msi because that's a dependency. Pytype fails tests due to https://github.com/google/pytype/issues/494 it looks like, I can change the hex to decimal if it's preferred to use decimal
I used `str` throughout the module instead of `path` because it didn't _look_ like the lower level code handles the paths correctly.

### CODE REVIEW

File: stdlib/2and3/msilib/__init__.pyi [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
+if sys.platform == 'win32':
+    from _msi import _Database
+
+    AMD64: bool
+    Itanium: bool
```
Comment (srittau) [inline stdlib/2and3/msilib/__init__.pyi]: This does not seem to exist in Python 3.8. (I did not test other versions.)

File: stdlib/2and3/msilib/__init__.pyi [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        fields: List[Tuple[int, str, int]]
+
+        def __init__(self, name: str) -> None: ...
+        def add_field(self, index: int, name: str, type: int) -> None: ...
+        def sql(self) -> str: ...
+        def create(self, db: _Database) -> None: ...
+
+    class _Unspecified: ...
+
+    def change_sequence(seq: List[Tuple[str, Optional[str], int]], action: str, seqno: Union[int, _Unspecified] = ..., cond: Union[str, _Unspecified] = ...) -> None: ...
```
Comment (srittau) [inline stdlib/2and3/msilib/__init__.pyi]: `Type[_Unspecified]` * 2 (needs import)

Also, `seq` can be any `Sequence`. (Also needs import.)

File: stdlib/2and3/msilib/__init__.pyi [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    def add_data(db: _Database, table: str, values: List[Tuple[Any, ...]]) -> None: ...
```
Comment (srittau) [inline stdlib/2and3/msilib/__init__.pyi]: `values` can be any `Iterable`. (Needs import.)

**Comment by srittau** [comment]
Also ping @rchen152 about the pytype build failure.

**Comment by CraftSpider** [comment]
Would it make sense to add 'deferred' for the pytype bug?

**Comment by CraftSpider** [comment]
Pytype has released a new version since the fix was merged, can the travis builds be re-run?

**Comment by srittau** [comment]
While the check here on GitHub did not get updated, the build now passed. Merging.

**Pull request merged**